test/users/test-delete.js

```diff
@@ -0,0 +1,17 @@
+const { assert } = require('chai');
+const { dispatch, mock_api, url_for, user } = require('../conftest');
+const handlers = require('../../app/users/handlers');
+
+
+describe('users', () => {
+  describe('delete', () => {
+    it('deletes user and redirects to list', () => {
+      mock_api().delete(`/users/${escape(user.auth0_id)}/`).reply(200, user);
```

Comment:
Not that it matters much in the test I think, but if I remember correctly `DELETE` endpoints returns [`204 No Content`](https://httpstatusdogs.com/204-no-content) on success.

Reply:
Too slow, I already changed that :)

Reply:
Ah :D

Reply:
Don't know why but feeling really slow this afternoon :P